corehq/apps/case_search/utils.py

```diff
@@ -328,11 +373,13 @@ def get_and_tag_related_cases(helper, app_id, case_types, cases,
     results = list({
         case.case_id: case for case in unfiltered_results if case.case_id not in initial_case_ids
     }.values())
-    for case in results:
-        _tag_is_related_case(case)
+    with helper.timing_context('_tag_is_related_case'):
+        for case in results:
+            _tag_is_related_case(case)
     return results
 
 
+@time_function()
 def get_related_cases_result(helper, app, case_types, source_cases, include_all_related_cases):
```

Comment:
This could either result in `_get_all_related_cases` OR `_get_search_detail_path_defined_cases` and `_get_child_cases_referenced_in_app`. Would it also be helpful to get the label the timing context for those parts?